### ISSUE TITLE
Slick dots ambiguous aria-label

### ISSUE THREAD

**Issue by akanksha-tripathi** [kenwheeler/slick#4270]
Issue :- Screen reader doest not read description label  of slick bullet points(slick-dots)

Impacted Version :- 1.8.1

Expected behaviour :- 
        1. The label of the tabs are describing the purpose of the tabs "Go to slide 1", "Go to slide 2", etc.
        2.Provide more descriptive text for the aria-label attribute for the tabs "Go to slide 1", "Go to slide 2", etc.
        3. By default I recommend to use "Previous slide" and "Next slide" as it's more understandable that just "Previous" and 
             "Next"

Observed behaviour :- 
        Label of the tags are announcing 1 of 8 which is rather confusing to user about the current state of slick dots or when user press on enter.

Recommendation:- 
        In earlier version 1.0.1 https://cdn.jsdelivr.net/npm/@accessible360/accessible-slick@1.0.1/slick/slick.min.js we had a class 
            "slick-sr-only" which had added aria-label as "Go to slide 1 of 2" which is no longer visible in new version of slick library

**Comment by akanksha-tripathi** [comment]
To bring to your notice about this issue that's been raised on July 25th, 2023. Do we have any update for the same

**Comment by wallaceb-dev** [comment]
Same issue here. There should be a way to customize this aria-label attribute when pluggin is initialized, considering that people may get confused listening to what the screen reader says.
